projects/angular-showcase/src/app/public/examples/autocomplete-showcase/autocomplete-showcase.component.html

```diff
@@ -16,7 +16,32 @@ <h4 class="sbbsc-block">Basic Example</h4>
 
   <div class="row">
     <div class="col-lg-6 sbbsc-v-sep">
-      <h4 class="sbbsc-block">Basic Example</h4>
+      <h4 class="sbbsc-block">Example Autocomplete Hint</h4>
+      <input type="text" [formControl]="myControlHint" [sbbAutocomplete]="autoHint" />
+      <sbb-autocomplete #autoHint="sbbAutocomplete">
+        <sbb-option
+          [value]="option"
+          *ngFor="
+            let option of filteredOptionsHint.length >= maxOptionsListLength
+              ? (filteredOptionsHint | slice: 0:maxOptionsListLength)
+              : filteredOptionsHint
+          "
+        >
+          {{ option }}
+        </sbb-option>
+        <sbb-autocomplete-hint *ngIf="filteredOptionsHint.length - maxOptionsListLength > 0">
```

Comment:
The ngIf can be rewritten to `filteredOptionsHint.length > maxOptionsListLength`, which is one computation less (this is very minor issue, but I thought I'd mention it)